### DISCUSSION TITLE
Cannot connect to the Vite Dev Server on http://localhost:5173

### DISCUSSION THREAD

**Discussion by josephthesde** [mubaidr/vite-vue3-chrome-extension-v3#99] [General]
Ran the latest master in my local(Mac OS), and got this issue in dev mode. Any idea?
Chrome Version 130.0.6723.70 (Official Build) (arm64)

<img width="230" alt="Screenshot 2024-10-26 at 11 39 06 PM" src="https://github.com/user-attachments/assets/8c7c31b0-d995-4557-9e38-8a97f1ca01a8">

**Comment by mubaidr** [comment]
Maybe the extension build process has not restarted yet? Please open developer page in chrome and reload extension.

**Comment by YourKey** [comment]
I have the same issue. During dev build, Vite is making requests to http://localhost:8888/ and thinks the dev server is not running, causing constant page reloads while trying to connect, even though the application actually opens at http://localhost:8888/src/popup/index.html#/popup.

**Comment by mubaidr** [comment]
A new update has been pushed just yesterday. Please try gain and report. I will test too.

**Comment by YourKey** [comment]
This issue appeared on its own, it wasn't present before, and I hadn't updated any dependencies.
Downgrading Vite to version 4.2.3 helped me fix it.
https://github.com/crxjs/chrome-extension-tools/issues/696

**Comment by mubaidr** [comment]
This is strange, I will check update th status.

**Comment by timczm** [comment]
I have the same issue on latest master with chrome 131.0.6778.70 & 127.0.6533.73 (Official Build) (64-bit) on windows.

**Comment by mubaidr** [comment]
You need to load this extension using: chrome://extensions/

Open above page, enable developer mode, then load extension from directory.

**Comment by mubaidr** [comment]
This issue exists in older versions of vite and crx tools plugin. Please use latest version for both. (beta in case of crx tools)
